sync/src/relayer/compact_block.rs

```diff
@@ -5,14 +5,11 @@ use ckb_protocol::{self, cast, FlatbuffersVectorIterator};
 use failure::Error as FailureError;
 use std::convert::{TryFrom, TryInto};
 
-pub type ShortTransactionID = [u8; 6];
-
 #[derive(Debug, PartialEq, Eq, Clone)]
 pub struct CompactBlock {
     pub header: Header,
     pub uncles: Vec<UncleBlock>,
-    pub nonce: u64,
-    pub short_ids: Vec<ShortTransactionID>,
+    pub short_ids: Vec<ProposalShortId>,
```

Comment:
I think the name of the `ProposalShortId` can now cause ambiguity. I think it should be replaced by `TransactionShortId`.

Reply:
keep the name as before pls.

Reply:
> I think the name of the `ProposalShortId` can now cause ambiguity. I think it should be replaced by `TransactionShortId`.

+1, I would like to do it in a sperate PR if this PR was accepted